packages/dds/tree/src/test/tableSchema.spec.ts

```diff
@@ -1617,95 +1621,5 @@ describe("TableFactory unit tests", () => {
 				// Respond to the change.
 			});
 		});
-
-		it("TableSchema: Remove column and corresponding cells in a transaction", () => {
```

Comment:
Note for reviewers: this example is no longer needed - this behavior is now captured directly by `table.removeColumns`.